opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -36,7 +36,7 @@
   /**
    * Like {@link #startTrace(String)}, but the returned span is made a child of {@code parent}.
    */
-  Span joinTrace(String operationName, TraceContext parent);
+  Span joinTrace(String operationName, SpanContext parent);
```

Comment:
I am fine with the name SpanContext. I am also fine with the name TraceContext. :)

No strong feelings, though perhaps a slight leaning towards TraceContext since it's a term that's used in multiple tracing systems already. From first principles, SpanContext seems more accurate as a descriptor, though.